src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemIPGlobalProperties.cs

```diff
@@ -194,7 +194,7 @@ private unsafe List<SystemTcpConnectionInformation> GetAllTcpConnections()
                             }
                         }
                     }
-                    catch
+                    finally
```

Comment:
Previously the `catch` would also unconditionally swallow the exception here; is that behaviour change a problem?

Reply:
We would not prior to #1914. So this matches original behavior and it is consistent with other cases @benaadams